flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -574,7 +588,7 @@ private void emitWindowContents(W window, ACC contents) throws Exception {
 	 *
 	 * @param element skipped late arriving element to side output
 	 */
-	private void sideOutput(StreamRecord<IN> element){
+	protected void sideOutput(StreamRecord<IN> element){
```

Comment:
Same as for `lateDataOutputTag`. It's used by the `EvictingWindowOperator`. I can also make it package private.